content/en/docs/concepts/architecture/master-node-communication.md

```diff
@@ -104,6 +104,17 @@ all traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the network in
 which the nodes are running.
 
-SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. A replacement for this communication channel is being designed.
+SSH tunnels are currently deprecated so you shouldn't opt to use them unless you know what you are doing. APIServer network proxy is a replacement for this communication channel.
+
+### APIServer network proxy (alpha)
```

Comment:
I recommend against putting feature state in links as they then aren't stable
```suggestion
### API Server network proxy
```
and add a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) for the section.

```suggestion
### API Server network proxy (alpha) {#apiserver-network-proxy}
```
also works, but I'm less keen.

Reply:
Didn't this make beta?